doc/basic_pv_pvc_dv.md

```diff
@@ -0,0 +1,219 @@
+# Storage
+
+Storage can be a tricky subject in kubernetes so we would like to give some basic guidance on how to implement storage for use with CDI. Storage in kubernetes is defined by Persistent Volumes ([PV](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)) which are a unit of storage. PVs define their size and other information needed to define the storage. When a user wants to use storage they define a Persistent Volume Claim ([PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)), which specifies the size of storage they want, and some other optional information. Kubernetes then tries to match an available PV to the PVC. If one is found, then the PVC is bound to the PV and the PVC becomes available. A pod can now mount the PVC using the pod spec when it starts.
+
+Cluster administrators can group PVs using storage classes. A [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is a way for administrators to describe the classes of storage they offer. You can specify which storage class to use when you request a PVC, this allows you to control which storage is used.
+
+In Containerized Data Importer (CDI) we have seen three types of storage being used for different use cases.
+* Manual hostPath based storage
+* local volume based storage
+* Dynamic provisioned storage
+
+Each one is useful for different use cases, which we will explain below. Note this document will likely become out of date as storage progresses at a rapid rate in kubernetes.
+
+### Manual storage
+We see this when someone just started with kubernetes and [kubevirt](https://github.com/kubevirt/kubevirt) and are using something like [minikube]( https://kubernetes.io/docs/setup/minikube/) or [minishift](https://github.com/minishift/minishift). These do not come preconfigured with storage and it is somewhat unclear on how to use local storage for some testing purposes. We will describe how to manually create a PV, and a matching PVC for use with CDI. Note a storage class is not defined nor required.
```

Comment:
So maybe now switch the doc up a bit... So instead of walking through these thing like this; instead just convert this to a section of "Example of how to use local storage in a Minikube deployment".

Then have a section that just points out that everything works the same if you already have a provisioner configured etc.